src/app.ts

```diff
@@ -94,6 +93,7 @@ const authenticatedRouter = new Router()
       ctx.status = 201;
       return;
     } catch (error) {
+      console.error(error);
```

Comment:
At some point we should set up a proper logger like [Winston](https://github.com/winstonjs/winston) or something but for now this is fine